docs/insets.md

```diff
@@ -202,6 +202,9 @@ For a more complex example, see the [`EdgeToEdgeLazyColumn`](https://github.com/
 
 ## Inset-aware layouts (`insets-ui`)
 
+!!! warning
+    **This library is deprecated, with [official support](https://android-review.git.corp.google.com/c/platform/frameworks/support/+/2667875) in androidx.compose.material.** The original documentation is below.
```

Comment:
Corp URL in OSS repo 😅

https://android-review.googlesource.com/c/platform/frameworks/support/+/2667875